src/PrestaShopBundle/Resources/config/services/adapter/address.yml

```
@@ -79,3 +79,6 @@ services:
     tags:
       - name: tactician.handler
         command: 'PrestaShop\PrestaShop\Core\Domain\Address\Command\EditCartAddressCommand'
+
+  prestashop.adapter.address.address_formatter:
+      class: 'PrestaShop\PrestaShop\Adapter\Address\AddressFormatter'
```

```suggestion
      class: 'PrestaShop\PrestaShop\Adapter\Address\Formatter'
```
Maybe no need to repeat the context in the class name since it's already defined in the namespace?